### PR TITLE
Preparations for Widget Deletion und Config

### PR DESCRIPTION
**Pull request by RudiKlassen**
* Preparations for Widget Delete and Config Functions (delete widget #181, edit widget #180)
* Deleted all html an css dependencies to info-box
* Also fixed a Layout problem with the divider widget

This is the new look of our widgets:
unbenannt-1

@matthiasbalke If we merge this changes, wie schould also edit the tutorial to add new widget, couse the structre of css has changed

### CODE REVIEW

**Comment by matthiasbalke** [comment]
I've took a look at the new design and attached a screenshot, with some widgets. I have to say, it looks like a lot of unused space. Maybe we can redesign the widget content. I like the menu bar for each widget, but it's consuming quite a lot of space.

**Comment by RudiKlassen** [comment]
I agree. We wasting al lot of space by giving each widget a header with custom buttons. To resolve that, we could possibly implement a fading effect by mouseover or implement a special edit mode where the cutom buttons for each widget show up. Let us discus this topic in our next session.
The current status could serve as template to implement the backend anyway.

**Comment by RudiKlassen** [comment]
Just pushed my new proposal for our new widgetlayout. It needs less space and inherits space for few icons(I added two icons/buttons. But they have no function yet). Let me know if it fits your ideas.

**Comment by RudiKlassen** [comment]
2017-03-27 21_31_26-infini board

**Comment by matthiasbalke** [comment]
Layout is looking awesome now!

File: dashy/src/app/dashboard/widget/jenkins/jenkins-widget.component.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.widget {
+  position: relative;
```
Comment (matthiasbalke) [inline dashy/src/app/dashboard/widget/jenkins/jenkins-widget.component.css]: You should move the css changes to `dashy/src/styles.css` so we don't have to apply them to every widget specific css file.

File: dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
+.widget {
+  position: relative;
```
Comment (matthiasbalke) [inline dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.css]: This is the same CSS as the widget above, right? Then you merge them as described in the comment above.

**Pull request merged**